pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,87 @@
+import NextAuth from 'next-auth'
```

Comment:
[Next-auth](https://next-auth.js.org/) handles our signin credentials for us. This file sets up the signin credentials to work with our MongoDB client.